Marille/Hub.cs

```diff
@@ -174,28 +197,35 @@ public async Task<bool> CreateAsync<T> (string topicName, TopicConfiguration con
 				return false;
 			}
 
-			var topicInfo = topic.CreateChannel (configuration, initialWorkers);
+			var topicInfo = topic.CreateChannel (configuration, errorWorker, initialWorkers);
 			await StartConsuming (topicInfo);
 			return true;
 		} finally {
 			semaphoreSlim.Release ( );
 		}
 	}
 
-	public Task<bool> CreateAsync<T> (string topicName, TopicConfiguration configuration,
+	public Task<bool> CreateAsync<T> (string topicName, TopicConfiguration configuration, IErrorWorker<T> errorWorker,
```

Comment:
Should these provide a default for errorWorker and errorAction so you don't break the API? Or leave the existing version of the functions and add additional overloads that add the errorWorker or errorAction?

Reply:
That is the issue I was mentioning about breaking the API, I could do something like provide a default that does nothing, throws the exception and crashes the process (or worse, gets lost). 

I bumped the version of the nuget precisely because of this.